src/main/java/com/se/homeworktwo/Sym.java

```diff
@@ -1,6 +1,54 @@
 package com.se.homeworktwo;
 
+import java.util.Map;
+import java.lang.Math;
+
 public class Sym
 {
-    // Add implementation for Sim
+    int n;
+    int at;
+    String name;
+    Map<String, Integer> _has;
+
+    public Sym(int col, String name) {
+        this.n = 0;
+        this._has.clear();
+        this.name = name;
+        this.at = col;
+    }
+
+    void add(String v) {
+        if (_has.containsKey(v)) {
```

Comment:
Should we add string null or Empty check for v?

Reply:
Makes sense, will add it